docs/administrator/security/certificate-configuration.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Certificate Configuration
+---
+
+## Certificate Configuration
+
+Karmada components use digital certificates to verify each other's identities, preventing malicious third parties from impersonating and stealing information or attacking the system. During mutual authentication with digital certificates between two components, the following files are involved:
+
+- Business Certificate (Client/Server): A certificate used to prove one's own identity.
+- Private Key: The private key corresponding to the public key included in the business certificate.
+- CA Root Certificate: The root certificate issued by the Certificate Authority (CA) that signed the business certificate.
+
+Both the business certificate and the CA root certificate have an expiration date. Therefore, when a business certificate or a CA root certificate is about to expire or has already expired, new certificates need to be replaced to ensure that the components can operate normally.
+
+### Prerequisites
+
+You need to install the following tools:
+
+- `openssl` ([Installation Guide](https://github.com/openssl/openssl#download))
+- `cfssl` ([Installation Guide](github.com/cloudflare/cfssl/cmd))
+- `cfssljson` ([Installation Guide](github.com/cloudflare/cfssl/cmd))
+
+
+
+> NOTE: The following examples are based on the installation method described in [Installation from Source](https://karmada.io/docs/installation/fromsource/#install). When using other installation methods, appropriate adaptation is required.
+
+### Karmada Certificates
+
+Karmada certificate storage path: Default to `${HOME}/.karmada`.
+
+The current certificates used by Karmada include:
+
+```shell
+$ tree
+.
+├── apiserver.crt
+├── apiserver.key
+├── ca-config.json
+├── ca.crt
+├── ca.key
+├── etcd-ca-config.json
+├── etcd-ca.crt
+├── etcd-ca.key
+├── etcd-client.crt
+├── etcd-client.key
+├── etcd-server.crt
+├── etcd-server.key
+├── front-proxy-ca-config.json
+├── front-proxy-ca.crt
+├── front-proxy-ca.key
+├── front-proxy-client.crt
+├── front-proxy-client.key
+├── karmada.crt
+└── karmada.key
+```
+
+#### **Karmada Certificate Overview**
+
+Certificates can be categorized into three sets based on the issuing CA:
+
+- Issued by ca

```

Comment:
```suggestion
- Issued by CA
```

Reply:
@samzong  `ca` is the name of a CA certificate, similar to `etcd-ca` and `front-proxy-ca`.

Reply:
How about:
```md
- Issued by CA certificate `ca`
```